docs/development_suite/api-console/api-design/request_limits.mdx

```diff
@@ -30,7 +30,9 @@ Once we get the basics of how it works, we can describe the possible combination
 
 - **No limits, no requests**: It is not recommended at all, because there are no controls on used resources and in case one pod has a high CPU demand that requires all the available CPU in the node, it will cause CPU starvation, rendering the other in a state of CPU throttling. Kubernetes will schedule this combination with the lowest QoS, `Best Effort`, meaning it will use all available resources. This pods will be the first to be evicted in case node pressure.
 - **No limits, with requests**: This case really depends on the performances requested to the pod, because you will be able to harness all available power, but risking cases of starvation. Kubernetes will schedule this combination with a `Burstable` QoS, meaning it will use all available resources in the node, but also ensuring that a lower-bound resource is guaranteed to the pod based on the request. This pods will be evicted after `Best Effort` ones.
-- **With limits, with or no requests**: Recommended for most cases, if requests isn't set it will automatically be set by Kubernetes as the same as limits. In this case you are sure that the pods are not stealing each other's resources, but you can have cases where excess cpu is not exploited, going to CPU throttling in case of high number of requests, even though it would not be necessary. If requests equals limits, Kubernetes will schedule this combination with a `Guaranteed` QoS, meaning that the pod has is guaranteed to receive the required resources, if such resources cannot be reserved, it will not scheduled. This pods will be the last to face eviction.
+- **With limits, with or no requests**: Recommended for most cases, if requests isn't set it will automatically be set by Kubernetes as the same as limits. In this case you are sure that the pods are not stealing each other's resources, but you can have cases where excess CPU is not exploited, going to CPU throttling in case of high number of requests, even though it would not be necessary. The QoS for this case can be:
+  - `Guaranteed` QoS, if requests equals limits and both explicitly defined in configuration, meaning that the pod is guaranteed to receive the required resources. If such resources cannot be reserved, it will not scheduled. This pods will be the last to face eviction;
+  - `Burstable` QoS in case only the limits it is explicitly defined.
```

Comment:
It is not the only case (and maybe it's a guaranteed case), read the comment above